### PR TITLE
fix: remove `long` types

### PR DESCRIPTION
**Pull request by SimenB**
If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

https://github.com/dcodeIO/long.js/pull/60 was included in v5

### CODE REVIEW

**Comment by SimenB** [comment]
Waiting for https://github.com/microsoft/DefinitelyTyped-tools/pull/513

File: types/bytebuffer/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/long": "^3.0.0"
```
Comment (SimenB) [inline types/bytebuffer/package.json:4]: https://github.com/protobufjs/bytebuffer.js/blob/4144951c7f583d9394d18487ff0b2b7474b1e775/package.json#L27

File: types/python-struct/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/long": "^4.0.0"
```
Comment (SimenB) [inline types/python-struct/package.json:4]: https://github.com/danielgindi/node-python-struct/blob/c1c72a00cbaac3d579939b4fcfae2333cf3528e5/package.json#L30

File: types/bytebuffer/package.json (hunk 1 above)
Comment (SimenB) [inline types/bytebuffer/package.json:4]: https://github.com/protobufjs/bytebuffer.js/blob/4144951c7f583d9394d18487ff0b2b7474b1e775/package.json#L27

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## bytebuffer ([<kbd>unpkg</kbd>](https://unpkg.com/browse/bytebuffer@latest/))
was missing the following properties:
1. Long
1. accessor
1. type
1. memcpy
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a16a9b79bb04cfce92c9d106f760eb0f400db626
</p>

**Review by peterblazejewicz — APPROVED** [review]
LGMT!
@SimenB thanks!

**Comment by SimenB** [comment]
Ready to merge

**Pull request merged**